src/popup/router/routes.js

```diff
@@ -261,6 +262,28 @@ export default [
       closeButton: true,
       hideNotificationsIcon: true,
     },
+  }, {
+    path: '/more/settings/networks/add',
+    name: 'network-add',
+    component: NetworkForm,
+    props: true,
+    meta: {
+      title: 'network-add',
+      backButton: true,
+      closeButton: true,
+      hideNotificationsIcon: true,
+    },
+  }, {
+    path: '/more/settings/networks/:name',
```

Comment:
What if someone will name its network "add" ? :D
I suggest to use following path:
```
'/more/settings/networks/edit/:name'
```